python/packages/nisar/workflows/stage_watermask.py

```diff
@@ -289,6 +291,20 @@ def translate_watermask(vrt_filename, outpath, x_min, x_max, y_min, y_max):
     input_x_min, xres, _, input_y_max, _, yres = ds.GetGeoTransform()
     length = ds.GetRasterBand(1).YSize
     width = ds.GetRasterBand(1).XSize
+
+    # Declare lambda function to snap min/max X and Y
+    # coordinates over the water grid
+    snap_coord = lambda val, snap, offset, round_func: round_func(  # noqa: E731
+        float(val - offset) / snap) * snap + offset
+
+    # Snap edge coordinates using the DEM pixel spacing
```

Comment:
There's no DEM here.

```suggestion
    # Snap edge coordinates using the raster pixel spacing
```